extensions.md

```diff
@@ -1515,6 +1504,21 @@ To alleviate this, consider implementing ```-webkit-overflow-scrolling: touch```
 ### Invoking an Extension with parameters (iOS)
 iOS support invocation URLS. An item can be linked to using these URLs.
 
+Example: [onmediafly://9cf282320e6340ee8b830e5376d54531product265958?param1=1&param2=2](onmediafly://9cf282320e6340ee8b830e5376d54531product265958?param1=1&param2=)
+
+This URL has the following format: [mcode]://[id]?[params...]
+
+The iOS app will pass all supplied params to the Extension and the Extension can
+retrieve them using JavaScript `window.location.search`.
+
+### Running the extension on iOS Next
```

Comment:
Might want to make it more generic than iOS next. This should be the "final" build process, in the future it should simply be just adding new commands. All platforms, ideally, will make their way to bootstrap.

Reply:
I thought we do this once we actually support bundled mflyCommands on at least one other platform. What do you suggest makes more sense to make it more generic?

Reply:
Now I see the steps to bootstrap are in `Building Extensions` so my previous comment can be ignored. I missed that before